test/k8sT/manifests/cilium-cm-patch.yaml

```diff
@@ -25,4 +25,5 @@ data:
   enable-ipv6: "true"
   preallocate-bpf-maps: "true"
   enable-legacy-services: "true"
-  bpf-ct-global-tcp-max: "1000000"
+  bpf-ct-global-tcp-max: "524288"
```

Comment:
we don't need this here as these are the default values picked from the default k8s descriptor.

Reply:
Do you mean that the values will be taken from the previously deployed ConfigMap?

Reply:
They should be taken from the configmap in `examples/kubernetes/x/cilium-cm.yaml`